### PR TITLE
Initial package structure

### PR DESCRIPTION
**Pull request by SamVanhoutte**
- created unit tests
- set up CI build
- initial package version

#3

### CODE REVIEW

File: src/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (tomkerkhove) [inline src/LICENSE:1]: We already have one in the root, is there a specific reason why you are adding it here as well?

File: src/readme.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Arcus for Azure Machine Learning
```
Comment (tomkerkhove) [inline src/readme.md:1]: I presume this README is to include it in the PyPi package?

File: src/LICENSE (hunk 1 above)
Comment (tomkerkhove) [reply src/LICENSE:1]: I presume this is to include it in the PyPi package?

If so, can we copy it as part of the build to avoid having 2 licenses?

File: build/ci-build.yml [hunk 3]
```diff
@@ -1,113 +1,56 @@
-name: $(date:yyyyMMdd)$(rev:.r)
+# Starter pipeline
 
-trigger:
```
Comment (tomkerkhove) [inline build/ci-build.yml:3]: You'll want to use this trigger instead of yours, it won't run against PRs

File: build/ci-build.yml [hunk 4]
```diff
@@ -1,113 +1,56 @@
-name: $(date:yyyyMMdd)$(rev:.r)
```
Comment (tomkerkhove) [inline build/ci-build.yml:1]: You'll want to use this build naming

File: build/ci-build.yml [hunk 5]
```diff
@@ -1,113 +1,56 @@
...
+
+- script: |    
+    python src/setup.py sdist 
+  displayName: 'Artifact creation'
+
+- task: CopyFiles@2  
+  inputs:    
+    targetFolder: $(Build.ArtifactStagingDirectory)
+
+- task: PublishBuildArtifacts@1
```
Comment (tomkerkhove) [inline build/ci-build.yml:52]: Is there a reason why we need this in CI? To verify the outcome?

File: README.md [hunk 6]
```diff
@@ -1,4 +1,19 @@
-# Arcus Template
-Template repository for new components in Arcus ecosystem.
+# Arcus - Azure Azure Machine Learning
+[![Build Status](https://dev.azure.com/codit/Arcus/_apis/build/status/Commit%20builds/CI%20-%20Arcus.AzureML?branchName=master)](https://dev.azure.com/codit/Arcus/_build/latest?definitionId=833&branchName=master)
+[![PyPI version](https://badge.fury.io/py/arcus-azure-ml.svg)](https://badge.fury.io/py/arcus-azure-ml)
 
-:warning: In order to finalize the setup, don't forget to run out [GitHub Repo Configurator](https://github.com/arcus-azure/arcus/tree/master/tools/github-repo-configurator).
+Azure Event Grid development in a breeze.
```
Comment (tomkerkhove) [inline README.md:5]: Can you update this please?

File: README.md [hunk 7]
```diff
@@ -1,4 +1,19 @@
...
+![Arcus](https://raw.githubusercontent.com/arcus-azure/arcus/master/media/arcus.png)
+
+# Installation
+
+```shell
+PM > pip3 install arcus-azure-ml
+```
+
+# Documentation
+All documentation can be found on [here](https://azureml.arcus-azure.net/).
```
Comment (tomkerkhove) [inline README.md:16]: This link doesn't work, do we still need to setup docs? If so, let's leave it out for now.

File: build/cd-build.yml [hunk 8]
```diff
@@ -0,0 +1,39 @@
+# Starter pipeline
```
Comment (tomkerkhove) [inline build/cd-build.yml:1]: As discussed, I would leave it out if it doesn't work yet and do a dedicated PR

File: build/cd-build.yml [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+- master
+
```
Comment (tomkerkhove) [inline build/cd-build.yml:9]: You'd want to disable this for PRs
```suggestion
pr: none
```

File: src/LICENSE (hunk 1 above)
Comment (SamVanhoutte) [reply src/LICENSE:1]: I removed it - don't see it being used anymore.

File: src/readme.md (hunk 2 above)
Comment (SamVanhoutte) [reply src/readme.md:1]: renamed to `package-description.md`just to be more clear

File: build/ci-build.yml (hunk 4 above)
Comment (SamVanhoutte) [reply build/ci-build.yml:1]: yes, took most of the standard template

File: build/ci-build.yml (hunk 3 above)
Comment (SamVanhoutte) [reply build/ci-build.yml:3]: yes, took most of the standard template

File: build/ci-build.yml (hunk 5 above)
Comment (SamVanhoutte) [reply build/ci-build.yml:52]: that was to have the build of the wheels effectively happening and being outputted as an artifact.  also believe we might publish these artifacts to a devops feed (like you publish to myget)

File: README.md (hunk 7 above)
Comment (SamVanhoutte) [reply README.md:16]: yep

File: build/cd-build.yml (hunk 8 above)
Comment (SamVanhoutte) [reply build/cd-build.yml:1]: removed it
